functest/operator_install_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright © 2020 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package functest
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	cmd "github.com/appsody/appsody/cmd"
+	cmdtest "github.com/appsody/appsody/cmd/cmdtest"
+	"github.com/pkg/errors"
+)
+
+func TestOperatorInstallCases(t *testing.T) {
+	var operatorInstallTests = []struct {
+		testName     string
+		args         []string
+		expectedLogs string
+	}{
+		{"Run in dryrun mode", []string{"--dryrun"}, "Appsody operator deployed to Kubernetes"},
```

Comment:
Not sure the expected log is the desired output for a `dryrun` flag

Reply:
I put that as the expected log since thats printed when the dryrun succeeds. I could either change this to check that no error has been received or just remove the test altogether if we're thinking of removing the dry run flag. Thoughts?